test/bases/renku_data_services/data_api/test_platform_config.py

```diff
@@ -48,3 +50,160 @@ async def test_patch_platform_config_unauthorized(
     _, res = await sanic_client.patch("/api/data/platform/config", headers=headers, json=payload)
 
     assert res.status_code == 403, res.text
+
+
+@pytest.mark.asyncio
+async def test_get_redirects(sanic_client: SanicASGITestClient, admin_headers: dict[str, str]) -> None:
+    parameters = {"page": 1, "per_page": 5}
+    headers = admin_headers
+    _, res = await sanic_client.get("/api/data/platform/redirects", headers=headers, params=parameters)
+
+    assert res.status_code == 200, f"status code {res.status_code} != 200"
+    assert res.json is not None
+    assert res.headers["page"] == "1"
+    assert res.headers["per-page"] == "5"
+    assert res.headers["total"] == "0"
+    assert res.headers["total-pages"] == "0"
+
+    # redirects = res.json
+    # assert redirects.get("etag") != ""
```

Comment:
reminder: remove or enable before merging

Reply:
Fixed in c98ae3fbf297010f5a183dda2b27a20576e4f932